### DISCUSSION TITLE
UNIVERSE DAY TWO: Who's ready?

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/events#58] [Keynotes]
We're just about an hour away from the second day of Universe 2022 🪐 . We're getting excited especially after a thrilling first day, see [what we're building](https://github.blog/2022-11-09-everything-new-from-github-universe-2022/) to enhance the most integrated developer platform that allows developers and enterprises to drive innovation with ease.

There is plenty more to come today, including but not limited to: 
* @abbycabs [spotlighting maintainers](https://githubuniverse.com/events/detail/virtual-schedule/d2b69f8a-c78c-4676-b609-649f69a932eb) at 10:30am PST
* GitHub, CSO and SVP @mph4 [shares how GitHub Builds GitHub](https://githubuniverse.com/events/detail/virtual-schedule/460787bd-08ed-4479-ae35-d3017457c813) at 11:00am PST
* @rmw's [GitHub Desktop is for Everyone](https://githubuniverse.com/events/detail/virtual-schedule/d98d21b8-2c51-4dc6-b) at 12:45 PST
* [10 Years of the Octoverse](https://githubuniverse.com/events/detail/virtual-schedule/c4a814e7-53cb-41d1-8) with GitHub VP @martinwoodward at 2:30 PST

Our [schedule](https://githubuniverse.com/events/detail/virtual-schedule) has you covered so you can plan for the day ahead 📆 . 

What session are you most excited for today 💥 ? Let us know ⬇️  in the comments!

**Comment by jricciardi** [comment]
Super excited for @abbycabs maintainers spotlight today! I really loved the similar chat yesterday with the OSS maintainers who are part of the Sponsors program 😄

**Comment by queenofcorgis** [comment]
I'm looking forward to more commentary and analysis from @anjuan, @filmgirl, and @damovisa!! They rocked it yesterday!

**Comment by ryandrewery** [comment]
Ready to listen to some great sessions from the community section 👍

**Comment by srinideverakonda** [comment]
looking forward to 
GitHub, CSO and SVP @mph4 [shares how GitHub Builds GitHub](https://githubuniverse.com/events/detail/virtual-schedule/460787bd-08ed-4479-ae35-d3017457c813) at 11:00am PST

**Comment by Aguimarcenes** [comment]
Let's Gooo!!!

**Comment by L0rdShrek** [comment]
👋

**Reply by imjohnbo** [reply]
👋 👀 😀

**Comment by Puri12** [comment]
I'm really looking forward to today's session😆

**Comment by ghostinhershell** [comment]
Let's goooo Day 2!!!! Really looking forward to @rmw's [GitHub Desktop is for Everyone](https://githubuniverse.com/events/detail/virtual-schedule/d98d21b8-2c51-4dc6-b) sesh 👀
